samples/gazell/gzll_ack_payload_device/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _gzll_ack_payload_device:
+
+Gazell ACK Payload Device
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows the basic Gazell communication and demonstrates how to send payloads and acknowledgments.
+It consists of two applications, one running on the Device and one running on the Host.
```

Comment:
The sample now is a single application after breaking down all gazell samples to the 4 individual ones. This particular sample runs Gazell in the _Device_ role.

Reply:
You are right, that is an important change that needs to be made. 
I have changed it to the following:
`This sample shows the Device role for basic Gazell communication.
As a single sample, the Device only sends packets and receives acknowledgements from the Host with a 1-byte payload.`